integreat_cms/core/signals/organization_signals.py

```diff
@@ -0,0 +1,20 @@
+"""
+This module contains signal handlers related to organization objects.
+"""
+
+from cacheops import invalidate_model
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from ...cms.models import Organization, PageTranslation
+
+
+@receiver(post_save, sender=Organization)
+def organization_create_handler(**kwargs):
+    r"""
+    Invalidate page translation cache after organization creation
+
+    :param \**kwargs: The supplied keyword arguments
+    :type \**kwargs: dict
+    """
+    invalidate_model(PageTranslation)
```

Comment:
```suggestion
    invalidate_model(Page)
    invalidate_model(PageTranslation)
```

You have to invalidate `Page`, because the organization field is stored in the page objects.
I tested with the children endpoint by changing the website field of an organization and it didn't work. After adding this line it worked.

Reply:
Oh. I tested with the specific page endpoint and there invalidating `PageTranslation` fixes the problem, but invalidating `Page` does not. I did not expect the behavior to differ between those endpoints and only tested the specific page endpoint. So to fix the problem completely, actually both models would need to be invalidated :/

Reply:
What.........Me neither have expected that 😅